app/services/compute_resource_host_importer.rb

```diff
@@ -0,0 +1,69 @@
+class ComputeResourceHostImporter
+  attr_accessor :host, :compute_resource, :vm
+
+  delegate :logger, :to => :Rails
+
+  def initialize(opts = {})
+    self.compute_resource = opts.fetch(:compute_resource)
+    self.vm = compute_resource.find_vm_by_uuid(opts[:uuid]) if opts[:uuid]
+    self.vm ||= opts.fetch(:vm)
+    self.host = Host::Managed.new(
+      :managed => true,
```

Comment:
When I import a host from hypervisor, I would expect the VM to be already built. Therefore I would expect `build => false` here. Otherwise the Host will be put in build mode after importing it (which risks deleting all data in the machine).